site-content/source/modules/ROOT/pages/blog.adoc

```diff
@@ -8,6 +8,30 @@ NOTES FOR CONTENT CREATORS
 - Replace post tile, date, description and link to you post.
 ////
 
+//start card
+[openblock,card shadow relative test]
+----
+[openblock,card-header]
+------
+[discrete]
+=== Farewell to Apache Cassandra 3.0.x and 3.11.x - End of Life Announcement 
```

Comment:
Probably just shorten the title for impact:
```suggestion
=== Farewell to Cassandra 3.0 and 3.11
```

Reply:
Good call. I'll re-edit that one.